README.md

```diff
@@ -515,6 +516,28 @@ Another alternative is [Chuck](https://github.com/jgilfelt/chuck) which, althoug
 
 [LeakCanary](https://github.com/square/leakcanary) is a library that makes runtime detection and identification of memory leaks a more routine part of your application development process. See the library [wiki](https://github.com/square/leakcanary/wiki) for details on configuration and usage. Just remember to configure only the "no-op" dependency in your release build!
 
+### Set up continuous integration
+
+Continuous integration systems let you automatically build and test your project every time you push in updates to version control. CI also runs static code analysis tools, generates the apk files and distributes them.
+Some popular code analysis tools are:
+
+* [Lint](https://developer.android.com/studio/write/lint.html)
+* [Checkstyle](http://checkstyle.sourceforge.net/)
+* [Danger](https://github.com/danger/danger)
+* [PMD](https://pmd.github.io/)
+* [Findbugs](http://findbugs.sourceforge.net/)
+   
```

Comment:
What about a quick mention to what each those static code analysis tools do. For instance, Danger - Help you to have a convention for your PRs in ur team.

Reply:
Also, would be nice to add here a quick reference to Fastlane to ship your app to the store automatically in beta?

Reply:
I would like to keep this as brief as possible, there are many tools and plugins that could be used. And I will follow Peter's suggestion about only recommending the ones I used.